### DISCUSSION TITLE
August 2021: taking a break · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#133] [Articles]
TL;DR: - this is a short article, no TL;DR.

**Comment by S1M0NH** [comment]
Health should always come first. Enjoy your break with your family.

**Comment by usermjs** [comment]
I can't speak for others, but I support you with a small token every month out of gratitude for your post explaining step-by-step how to set up OpenSMTPD.  It was exactly what I needed to set up what to me is a perfect system for email.  

Reading these personal posts of yours, it sounds to me like you are too hard on yourself.  You have done great things with your talents already, and will surely do more for the joy of it as time permits in the future.  Do it when and because you want to, not when and because you think you "should."

Merci

**Comment by terminar** [comment]
Hey Gilles (@poolpOrg),

first of all, of course stay healthy, that's the most important part.

Regarding Plakar: sounds like an interesting project and program - if you need some alpha/beta testers just give me a msg, I'm in.
But reading that comment:
_I realized it wasn’t and didn’t bring any benefit over creating multiple plakars. I decided to remove namespaces to simplify things._ 

I have some feedback. I think you are maybe right looking at Plakar from a single user perspective who has complete control over the (remote) server. But If you are starting to think about something like hosting Plakar instances that may be a problem. It doesn't make sense to create different Plakar instances (listening on a tons of different ports) for every new "repository". Maybe I've just not understood the principle of Plakar yet.
The Plakar server (and proxy server) stuff is really great. Will there be some user management implemented?

Bye
Björn

**Comment by poolpOrg** [comment]
Hi @terminar,

Behind a running plakar instance you have a plakar repository and my initial take was that this repository could contain multiple namespaces. My new take is that if you need multiple namespaces you might as well create multiple repositories instead and in a multi-user setup, it will be the goal of the server to back the session with the proper repository so that only one port is bound.

At this point, the network part is mostly a poc so that I could make sure the primitives I came up with would work on a remote machine, there's still work to do in that area.

I have not touched the code since my last article but I have plans to spend a day or two working on plakar in October.

I'll let you know when you can alpha/beta :-)

**Reply by poolpOrg** [reply]
There you go @terminar :

https://github.com/poolpOrg/plakar

**Comment by S1M0NH** [comment]
Hi Gilles. When you get time, would you mind writing a post covering common setups, maybe in FAQ format. Something like this:

1. How do I setup an outgoing only server
2. How do I setup to use another smtp server (i.e. gmail, outlook etc.)
3. How do I setup a mailing list
4. How do I receive mail
5. How can I have mail forwarded somewhere else

It could be the go to place for questions.
